modules/javafx.controls/src/test/java/test/javafx/scene/control/ToolbarTest.java

```diff
@@ -58,16 +63,30 @@
 public class ToolbarTest {
     private ToolBar toolBar;//Empty
     private ToolBar toolBarWithItems;//Items
+
     private Toolkit tk;
+    private Scene scene;
+    private Stage stage;
+    private StackPane root;
+
     private Node node1;
     private Node node2;
 
+    private static final double TOOLBAR_SIZE = 300.0;
+    private static final double EXPANDED_CHILDREN_SIZE = 250.0;
+    private static final double ORIGINAL_CHILDREN_SIZE = 100.0;
+
     @Before public void setup() {
         tk = (StubToolkit)Toolkit.getToolkit();//This step is not needed (Just to make sure StubToolkit is loaded into VM)
         toolBar = new ToolBar();
         node1 = new Rectangle();
         node2 = new Rectangle(2.0,4.0);
         toolBarWithItems = new ToolBar(node1,node2);
+
+        root = new StackPane();
+        scene = new Scene(root);
```

Comment:
scene & stage creation + stage.setScene() can be moved to private method initializeToolBar(), Also we need not have them as class members.

Reply:
Thanks for making the change.
You have not addressed the second half my earlier comment - scene and stage need not be class members.

Reply:
Have you missed fixing this?
I am OK with the changes and will approve once you fix this comment.